Library/Formula/anttweakbar.rb

```diff
@@ -64,3 +63,16 @@ def install
          }
          if (gl_dyld)
          {
+diff --git a/src/Makefile.osx b/src/Makefile.osx
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!